### DISCUSSION TITLE
About PHP version 8.x

### DISCUSSION THREAD

**Discussion by Cedricocorico** [nilsteampassnet/TeamPass#3605] [Q&A]
Helle everybody i have quick question about a fresh install.
In requirement for the PHP extensions it is write to have mcrypt but it was delete since PHP 7.3 (and has been removed from PHP 7.2) and they are other requirement that don't appear for PHP Extension in WampServer. So I don't know if I can Install TeamPass v3. If someone has make a fresh install i will be glad to know how did you do it.

**Comment by jofrey007** [comment]
I did fresh installation on Debian 11

root@teampass:~# cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye


apt install php8.1
apt install php8.1-bcmath
apt install php8.1-mcrypt
apt install php8.1-mbstring
apt install php8.1-openssl
apt install php8.1-iconv
apt install php8.1-gd
apt install php8.1-mysql
apt install php8.1-xml
apt install php8.1-gmp
apt install php8.1-curl
apt install php8.1-ldap

**Reply by Cedricocorico** [reply]
Thanks for the reply. I will try on Debian 11, did you follow this doc for your installation ? : https://teampass.readthedocs.io/en/latest/install/install-linux/

**Reply by jofrey007** [reply]
This is installation steps for version 2.X

I followed up this installation of Teampass

https://documentation.teampass.net/#/?id=main

And LAMP for Debian 11 here

https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mariadb-php-lamp-stack-on-debian-11

**Comment by nilsteampassnet** [comment]
@Cedricocorico 
Yes you need php extensions listed in https://documentation.teampass.net/#/install/installation?id=install-the-apache-web-server-and-the-required-php-extensions

**Comment by Cedricocorico** [comment]
Thanks for all the help i have been able to install TeamPass on Debian even if i use php 7.4.
